src/neo/SmartContract/ApplicationEngine.Native.cs

```diff
@@ -6,24 +6,24 @@ namespace Neo.SmartContract
 {
     partial class ApplicationEngine
     {
-        public static readonly InteropDescriptor Neo_Native_Deploy = Register("Neo.Native.Deploy", nameof(DeployNativeContracts), 0, CallFlags.AllowModifyStates, false);
+        public static readonly InteropDescriptor Neo_Native_Deploy = Register("Neo.Native.Deploy", nameof(DeployNativeContract), 0, CallFlags.AllowModifyStates, false);
```

Comment:
Do we need `Neo.Native.Update` ? How do we update native contract?

Reply:
I think that we should do a condition inside the method, and change the method hash in order to include the arguments, in this way we can have method overloads.